lib/krane/container_logs.rb

```diff
@@ -3,7 +3,9 @@ module Krane
   class ContainerLogs
     attr_reader :lines, :container_name
 
-    DEFAULT_LINE_LIMIT = 25
+    def self.default_line_limit
+      Integer(ENV["LOG_LINE_LIMIT"], exception: false) || 25
```

Comment:
```suggestion
      Integer(ENV.fetch('KRANE_LOG_LINE_LIMIT', 25))
```

Reply:
` Integer(ENV.fetch('KRANE_LOG_LINE_LIMIT', 25))` would throw an exception if it was a string that can't be converted into an integer (eg. `ENV['KRANE_LOG_LINE_LIMIT']=lol`, are we fine with letting that happen since we're probably going to be the only ones using it?

Reply:
Yes I know. But if you put garbage in that ENV var, we might as well let you know rather than silently getting back to the default.